xdsl/interpreter.py

```diff
@@ -252,7 +263,35 @@ def register_implementations(
         """
         self._impls.register_from(impls, override=override)
 
-    def run(self, op: Operation):
+    def _fetch_direct_symbol(self, op: Operation, symbol: StringAttr):
```

Comment:
I guess this part is going to be changed with #1146

Reply:
Indeed, didn't want to steal it nor wait!